src/app/shared/components/notification-container/notification-container.component.ts

```diff
@@ -0,0 +1,147 @@
+import { AfterViewInit, ApplicationRef, Component, ComponentRef, createComponent, ElementRef, EmbeddedViewRef, Input, OnDestroy, OnInit } from '@angular/core'
+import { NotificationService } from '../../services/notification.service'
+import { Subject, takeUntil } from 'rxjs'
+import { NotificationComponent } from '../notification/notification.component'
+import { Notification } from '../../models/notification'
+import { StatusMessageEventObserver } from '../../../core/services/status-message-event-observer'
+import { EventSubscription } from '../../../event-system/abstractions/event-observer.service'
+import { StatusMessage } from '../../models/status-message'
+import { StatusCode } from '../../enums/status-code'
+import { SystemInformationService } from '../../services/system-information.service'
+
+const NOTIFICATION_DURATION_MS: number = 7000
+const TOP_PADDING: number = 5
+const PIXEL_POSTFIX: string = 'px'
+
+const STACK_GAP: number = 5
+
+const NON_PERSISTENT_NOTIFICATION_ID: string = 'NON_PERSISTENT_NOTIFICATION_ID'
+
+@Component({
+  selector: 'sofie-notification-container',
+  templateUrl: './notification-container.component.html',
+  styleUrls: ['./notification-container.component.scss'],
+})
+export class NotificationContainerComponent implements OnInit, OnDestroy, AfterViewInit {
+  @Input()
+  public topOffset: number = 0
+
+  @Input()
+  public rightOffset: number = 0
+
+  private readonly destroySubject: Subject<void> = new Subject()
```

Comment:
This sounds like something that can be called. Suggestions: `componentLifespanSubject` or `lifecycleSubject`

Reply:
To me that sounds like a Subject that will emit everytime one of the lifecycle methods is being called, which is not the case. This Subject will only emit once the Component is about to be destroyed.

Reply:
I saw Viktor had a good name for it. He called it `unsubscribe$`. I'm going to rename it to that